lib/class-wp-rest-image-editor-controller.php

```diff
@@ -101,90 +101,134 @@ public function permission_callback( $request ) {
 	public function apply_edits( $request ) {
 		require_once ABSPATH . 'wp-admin/includes/image.php';
 
-		$params = $request->get_params();
+		$params        = $request->get_params();
+		$attachment_id = $params['media_id'];
 
-		$media_id = $params['media_id'];
+		// This also confirms the attachment is an image.
+		$image_file = wp_get_original_image_path( $attachment_id );
```

Comment:
Might be good to leave a comment saying that this is the original uploaded file.